fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/SimulationFacade.java

```diff
@@ -382,9 +382,21 @@ public TraciSimulationStepResult simulateUntil(long time) throws InternalFederat
                         log.info("Vehicle {} has parked at {} (edge: {})", veh.id, veh.position, veh.edgeId);
                     }
                     vehicleData = new VehicleData.Builder(time, lastVehicleData.getName())
-                            .copyFrom(lastVehicleData)
                             .position(veh.position.getGeographicPosition(), veh.position.getProjectedPosition())
-                            .stopped(vehicleStopMode).create();
+                            .road(lastVehicleData.getRoadPosition())
```

Comment:
But isn't that at the wrong position then?

Reply:
Yes and no, the issue is that if we update the road position while the vehicle is parked, we get negative lane offsets, which are differently asserted in our tests. We would need to adjust our tests, if we properly update this...

Reply:
ok, then better keep it as it is, otherwise we would brake routing I guess

Reply:
Yes, that's what I figured aswell